equational_theories/AllEquations.lean

```diff
@@ -1,6 +1,7 @@
 import equational_theories.Magma
 import equational_theories.Equations
 import equational_theories.EquationsCommand
+import equational_theories.EquationalResult
```

Comment:
That doesn’t seem to be needed for this file, does it? Generally maybe better to add it in the files that actually need this, to keep the module graph more parallel, and that you don’t have to recompile `AllEquations` just because the attribute code changes.

Reply:
It was quite slow to compile `AllEquations`, but maybe it got faster. 🤷🏻